### PR TITLE
add documentation on slurm job control

### PR DESCRIPTION
**Pull request by PrometheusPi**
This adds some examples to the slurm documentation explaining how to control your job after submission to the scheduler.

### CODE REVIEW

File: src/picongpu/submit/taurus-tud/Slurm_Tutorial.rst [hunk 1]
```diff
@@ -36,4 +36,7 @@ Job Control
 
   * `scancel 12345` abort job
   * `scancel -s Number 12345` send signal or signal name to job
-  * `scontrol update timelimit=4:00:00 jobid=12345`
+  * `scontrol update timelimit=4:00:00 jobid=12345` change the walltime of the job
+  * `scontrol update jobid=12345  dependency=afterany:54321` only start the job after `54321` has finished
+  * `scontrol hold jobid=12345 ` prevent the job from staring
```
Comment (ax3l) [inline src/picongpu/submit/taurus-tud/Slurm_Tutorial.rst]: starting

File: src/picongpu/submit/taurus-tud/Slurm_Tutorial.rst [hunk 2]
```diff
@@ -36,4 +36,7 @@ Job Control
 
   * `scancel 12345` abort job
   * `scancel -s Number 12345` send signal or signal name to job
-  * `scontrol update timelimit=4:00:00 jobid=12345`
+  * `scontrol update timelimit=4:00:00 jobid=12345` change the walltime of the job
+  * `scontrol update jobid=12345  dependency=afterany:54321` only start the job after `54321` has finished
```
Comment (ax3l) [inline src/picongpu/submit/taurus-tud/Slurm_Tutorial.rst]: double whitespace

Comment (ax3l) [inline src/picongpu/submit/taurus-tud/Slurm_Tutorial.rst]: ... the job after job with id `54321` has finished

File: src/picongpu/submit/taurus-tud/Slurm_Tutorial.rst [hunk 3]
```diff
@@ -36,4 +36,7 @@ Job Control
...
+  * `scontrol hold jobid=12345 ` prevent the job from staring
+  * `scontrol unhold jobid=12345 ` allow the job to start (after a hold was set)
```
Comment (ax3l) [inline src/picongpu/submit/taurus-tud/Slurm_Tutorial.rst]: space before `'`

Comment (ax3l) [inline src/picongpu/submit/taurus-tud/Slurm_Tutorial.rst]: change comment to: release the job to be eligible for run (after it was set on hold)

**Comment by PrometheusPi** [comment]
@ax3l thanks for the review. I fixed all mentioned points.

**Pull request merged**